README.md

```diff
@@ -63,8 +63,15 @@ Notes about using other rubies. If you don't use rvm, you can probably skip
 this paragraph. Logstash works with other rubies, and if you wish to use your
 own ruby you must set `USE_RUBY=1` in your environment.
 
-We recommend using flatland/drip for faster startup times during development. To
-tell Logstash to use drip, set `USE_DRIP=1` in your environment.
+## Drip Launcher
+
+[Drip](https://github.com/ninjudd/drip) is a launcher for the Java Virtual Machine that provides much faster startup times than the `java` command. The drip script is intended to be a drop-in replacement for the java command, only faster. We recommend using drip during development. 
+
+To tell Logstash to use drip, either set the `USE_DRIP=1` environment variable or set `JAVACMD=which drip`.
```

Comment:
the javacmd should be set like

```
JAVACMD=`which drip`
```

I am not sure how to escape the back-tick in markdown